server/config/toml.go

```diff
@@ -16,7 +16,7 @@ const defaultConfigTemplate = `# This is a TOML config file.
 # The minimum gas prices a validator is willing to accept for processing a
 # transaction. A transaction's fees must meet the minimum of each denomination
 # specified in this config (e.g. 0.01photino,0.0001stake).
-minimum_gas_prices = "{{ .BaseConfig.MinGasPrices }}"
+minimum-gas-prices = "{{ .BaseConfig.MinGasPrices }}"
```

Comment:
So is this required? This is why I didn't go with this in the first place because most (all?) toml keys are snake-cased.

Reply:
That is true for Tendermint `config.toml`, not for `gaiad.toml`.

Reply:
You're right...TOML spec doesn't really seem to have a preference here (https://github.com/toml-lang/toml#user-content-keys)